src/docs/product/integrations/integration-platform/ui-components/index.mdx

```diff
@@ -5,348 +5,40 @@ redirect_from:
 description: "Learn more about Sentry's Integration Platform User Interface (UI) components and how they provide developers with the ability to define UI components to render within specific parts of Sentry's UI, through a JSON-based schema."
 ---
 
-The Sentry Integration Platform provides developers with the ability to define User Interface components to render within specific parts of Sentry's UI, through a JSON-based schema. Two possible use-cases are [Issue Link](#issue-link) and [Stack Trace Link](#stack-trace-link).
+The Sentry Integration Platform provides developers with the ability to define components to render within specific parts of [sentry.io](https://sentry.io), through a JSON-based schema.
 
-# Schema
+You can define the following components:
+ - [Issue Link](/product/integrations/integration-platform/ui-components/issue-link) (uses FormField components)
+ - [Alert Rule Actions](/product/integrations/integration-platform/ui-components/alert-rule-action) (uses FormField components)
+ - [Stack Trace Link](/product/integrations/integration-platform/ui-components/stacktrace-link)
```

Comment:
Looks great! Could you hyperlink-ify the occurances of `FormField` though, since now the only way to get to your great new page is through the wonky side bar which doesn't autoscroll